srcpkgs/etlegacy/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'etlegacy'
+pkgname=etlegacy
+version=2.77.1
+revision=1
+archs="arm* i686 x86_64*"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DCMAKE_LIBRARY_PATH=/usr \
+ -DCMAKE_INSTALL_PREFIX=/usr -DINSTALL_DEFAULT_BASEDIR=/usr/lib/etlegacy \
+ -DCLIENT_GLVND=1 -DBUNDLED_LIBS=0 -DFEATURE_AUTOUPDATE=0 \
+ -DINSTALL_OMNIBOT=0"
+hostmakedepends="libtool pkg-config"
+makedepends="libglvnd-devel glew-devel SDL2-devel zlib-devel minizip-devel
+ libjpeg-turbo-devel libcurl-devel lua54-devel libvorbis-devel libtheora-devel freetype-devel
+ libpng-devel sqlite-devel libopenal-devel"
+short_desc="Project based on Wolfenstein: Enemy Territory"
+maintainer="biopsin <biopsin@teknik.io>"
+license="GPL-3.0-or-later"
+homepage="https://www.etlegacy.com"
+changelog="https://github.com/etlegacy/etlegacy/wiki/Changelog"
+distfiles="https://github.com/etlegacy/etlegacy/archive/refs/tags/v${version}.tar.gz"
+checksum=730a8a52435884b922d8c280fcdeace648902399798c7a973da72fbc4163ebe2
+
+if [ "$CROSS_BUILD" ] || [ "$XBPS_TARGET_WORDSIZE" != 32 ]; then
```

Comment:
remove

Reply:
Sorry, the -DCROSS_COMPILE32 flag has me completely confused.
Pushed again omiting flag.

Reply:
It defaults to 32-bit without -DCROSS_COMPILE32=0

Reply:
then always use -DCROSS_COMPILE32 without any condition